pkg/render/logstorage.go

```diff
@@ -138,6 +138,39 @@ const (
 )
 
 const (
+	// ElasticsearchKeystoreSecret Currently only used when FIPS mode is enabled, we need to initialize the keystore with a password.
+	ElasticsearchKeystoreSecret         = "tigera-secure-elasticsearch-keystore"
+	ElasticsearchKeystoreEnvName        = "KEYSTORE_PASSWORD"
+	ElasticsearchKeystoreHashAnnotation = "hash.operator.tigera.io/keystore-password"
+
+	keystoreInitContainerName = "elastic-internal-init-jvm-keystore"
+	keystoreInitScript        = `#!/usr/bin/env bash
```

Comment:
Move this script template into a template file and use [go embed](https://pkg.go.dev/embed)?